tools/schemacode/bidsschematools/validator.py

```diff
@@ -514,7 +514,7 @@ def select_schema_dir(
         according to the respective schema version, e.g. "1.7.0".
         If the path starts with the string "{module_path}" it will be expanded relative to the
         module path.
-    schema_version : str or None
+    schema_version : str | None
```

Comment:
Above you went in the other direction.

Reply:
Ah missed this one- it should be an `or`. From what I saw from numpydoc that's what's used